types/proto/narwhal.proto

```diff
@@ -96,8 +96,8 @@ message CollectionRetrievalResult {
     }
 }
 
-message GetCollectionsRequest {
-    // List of collections to be retreived.
+message Collections {
```

Comment:
I definitely see the point of re-using a message for both functionalities, I am just a bit skeptical on the convention though. Since this is a _message_ the `Collections` on its own doesn't mean much to be honest and in most examples I've seen conventions closer to e.x `GetCollectionsRequest` , `RemoveCollectionsRequest`. I believe keeping those separate might benefit us for future evolution (in case those differentiate as well)

Reply:
You are correct `GetCollectionRequest`/`RemoveCollectionRequest` is the convention. I will switch back to that format instead of reusing the `Collections` message.